CHANGELOG.md

```diff
@@ -4,11 +4,24 @@ As of now, while we are still on version `0.*`,
 we do not use [Semantic Versioning](https://semver.org/).
 Specifically this means, that minor upgrades can contain **breaking changes**.
 
-# Unreleased yet
-* Bugfix: Write undeploy versions correctly into manifest repo
-
 # Change Log
 
+## 0.4.51
+* Bugfix: Write undeploy versions correctly into manifest repo [#412](https://github.com/freiheit-com/kuberpult/pull/412)
+* route /v2/home to index.html in build directory [#454](https://github.com/freiheit-com/kuberpult/pull/454)
+* Fix exec planner [#453](https://github.com/freiheit-com/kuberpult/pull/454)
```

Comment:
The PR is wrong here, also don't need to include fix exec planner, this is something internal

Reply:
fixed!